tools/esdlcmd/esdl-publish.cpp

```diff
@@ -615,7 +599,7 @@ class EsdlDeleteESDLDefCmd : public EsdlPublishCmdCommon
                     optESDLService.set(arg);
                     break;
                 case 1:
-                    optVersionStr.set(arg);
+                    optDefinitionVersionStr.set(arg);
```

Comment:
Agreed, my original implementation introduced this inconsistency. Changing things now will break users, but we might have to tear off the band-aid quickly and move on. 
If we make the change, we could make the fix backward compatible, but I'm not sure if that would be worth the effort.